### ISSUE TITLE
Pagination for colored runs not working correctly

### ISSUE THREAD

**Issue by vdbergh** [glinscott/fishtest#1224]
Still trying to wrap my head around pagination in Fishtest.

If we view for example yellow runs, Fishtest currently list 4618 page but from page 509 on, all pages are empty E.g. https://dfts-0.pigazzini.it/tests/finished?page=509&yellow_only=1 .

The culprit I think is the following code
```python
            count = self.runs.estimated_document_count(hint=idx_hint)
```
`estimated_document_count` does not seem to take a `hint` argument. See https://docs.mongodb.com/manual/reference/method/db.collection.estimatedDocumentCount/

The correct way to handle this IMHO is to create a "yellow" view on the "runs" collections and then to use `estimated_documment_count` on this view. See https://docs.mongodb.com/manual/core/views/.

**Comment by vondele** [comment]
eventually it would be nice if we could somehow merge and make more flexible the list of tests displayed. Like right now, I sometimes would like to see all LTC (also active), or my tests, that are LTC, etc.

**Comment by vdbergh** [comment]
> eventually it would be nice if we could somehow merge and make more flexible the list of tests displayed. Like right now, I sometimes would like to see all LTC (also active), or my tests, that are LTC, etc.

The api almost supports this already. https://tests.stockfishchess.org/tests/user/vdv?ltc_only=1 . Currently it is (or should be) restricted to finish runs, but that is easily fixed I think. However the pagination is broken (4622 pages most of which are empty). This is not a practical problem, but it looks very unprofessional.

**Comment by ppigazzini** [comment]
Joost has 115528 finished LTC ???

**Comment by vdbergh** [comment]
More criteria are also possible

https://tests.stockfishchess.org/tests/user/vdv?ltc_only=1&success_only=1

All finished green LTC tests by vondele.

**Comment by vdbergh** [comment]
Fixed by #1227 .
